packages/askui-nodejs/example_projects_templates/typescript_jest/test/helper/jest.setup.ts

```diff
@@ -10,12 +12,29 @@ let aui: UiControlClient;
 jest.setTimeout(60 * 1000 * 60);
 
 beforeAll(async () => {
+  const proxyAgents = {};
+  /*
+  // Proxy configuration
+  // Uncomment this block for proxy support.
+  const proxyUrl = "http://<your-proxy-url>"
+  proxyAgents = {
+    proxyAgents: {
+      http: new HttpProxyAgent({
+          proxy: proxyUrl
+        }),
+      https: new HttpsProxyAgent({
+          proxy: proxyUrl
+        }),
+  }}
+  */
```

Comment:
Just an idea of how we might make this a little cleaner: what about moving the proxy stuff to a file right next to `jest.setup.ts`, e.g., `proxies.ts`, that exports a function that gets the url as a parameter and returns the proxy agents?

We could even use dynamic imports inside that function testing if `hpagent` is available logging something out if not, throwing an error if at the same time an url was provided with a hint to install it etc.

Reply:
How can I use dynamic imports? `await import('hpagent')`

I was also playing around, with the idea to use the proxy from the `HTTP_PROXY` env variables

Reply:
Like the idea :) would be nice to be consistent in how we configure stuff, so maybe for the other stuff too; your use of dynamic `import` seems right (see https://v8.dev/features/dynamic-import)